# Two square brackets closed but only one opened
#
# Looked it up, new closing square bracket should have opening square bracket for whole 19. gr. a.
# See 19. gr. a, https://www.althingi.is/lagas/151c/1995134.html
#     + 1) https://www.althingi.is/altext/stjt/2021.018.html
#     + 2) https://www.althingi.is/altext/stjt/2005.062.html#G13
--- cleaned/1995-134.html	2022-01-21 18:16:00.690116352 +0000
+++ patched/1995-134.html	2022-01-21 19:06:13.395058432 +0000
@@ -1101 +1101 @@
-   [19. gr. a.
+   [[19. gr. a.
